File: PY/Petshop.py
class Petshop:
    __id = ""
    __nama = ""
    __kategori = ""
    __harga = 0
    
    
    def __init__(self, id = "", nama = "", kategori = "", harga = 0):
        self.__id = id
        self.__nama = nama
        self.__kategori = kategori
        self.__harga = harga
    
    def updateAll(self, id = "", nama = "", kategori = "", harga = 0):
        self.__id = id
        self.__nama = nama
        self.__kategori = kategori
        self.__harga = harga
    
    def getId(self):
        return self.__id
    
    def setId(self, id):
        self.__id = id
    
    def getNama(self):
        return self.__nama
    
    def setNama(self, nama):
        self.__nama = nama
    
    def getKategori(self):
        return self.__kategori
    
    def setKategori(self, kategori):
        self.__kategori = kategori
    
    def getHarga(self):
        return self.__harga
    
    def setHarga(self, harga):
        self.__harga = harga
    

File: PHP/Petshop.php
<?php
    class Petshop
    {
        private $id;
        private $nama;
        private $kategori;
        private $harga;
        private $gambar;

        public function __construct($id="", $nama="", $kategori="", $harga="", $gambar){
            $this->id = $id;
            $this->nama = $nama;
            $this->kategori = $kategori;
            $this->harga = $harga;
            $this->gambar = $gambar;
        }

        public function setId($id){
            $this->id = $id;
        }
        public function getId(){
            return $this->id;
        }
        public function setNama($nama){
            $this->nama = $nama;
        }
        public function getNama(){
            return $this->nama;
        }
        public function setKategori($kategori){
            $this->kategori = $kategori;
        }
        public function getKategori(){
            return $this->kategori;
        }
        public function setHarga($harga){
            $this->harga = $harga;
        }
        public function getHarga(){
            return $this->harga;
        }
        public function setGambar($gambar){
            $this->gambar = $gambar;
        }
        public function getGambar(){
            return $this->gambar;
        }
    }
?>